src/main/kotlin/ee/carlrobert/codegpt/codecompletions/CodeCompletionRequestFactory.kt

```diff
@@ -114,14 +114,21 @@ object CodeCompletionRequestFactory {
 
     fun buildOllamaRequest(details: InfillRequest): OllamaCompletionRequest {
         val settings = service<OllamaSettings>().state
+        val prompt = if (details.context == null || details.context.contextElements.isEmpty()) {
```

Comment:
At the moment, the `details.context` will always be null and it is not configurable from settings either.